sale_order_delivery_date/models/sale_order_line.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Quartile Limited
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+from pytz import UTC, timezone
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.multi
+    def _prepare_procurement_values(self, group_id=False):
+        values = super(SaleOrderLine, self)._prepare_procurement_values(group_id)
+        self.ensure_one()
```

Comment:
```suggestion
        self.ensure_one()
        values = super(SaleOrderLine, self)._prepare_procurement_values(group_id)
```
Or remove `self.ensure_one()` perhaps?